event_flow/__init__.py

```diff
@@ -0,0 +1,17 @@
+import inspect
+from typing import TypeVar
+
+T = TypeVar("T")
+
+
+def export(definition: type[T]) -> type[T]:
```

Comment:
Maybe it's better to move `export` out of the main package init? I don't think it is needed outside event-flow

Reply:
Moved to the utils package.